apps/builder/app/builder/shared/commands.ts

```diff
@@ -134,6 +134,22 @@ export const { emitCommand, subscribeCommands } = createCommandsEmitter({
       },
       disableHotkeyOnFormTags: true,
     },
+    {
+      name: "openStylePanel",
+      defaultHotkeys: ["s"],
+      handler: () => {
+        $activeInspectorPanel.set("style");
+      },
+      disableHotkeyOnFormTags: true,
```

Comment:
need to forbid in content editable as well
now I cannot enter s or d when rename instances